api/src/main/java/com/openlattice/edm/requests/MetadataUpdate.java

```diff
@@ -73,14 +78,14 @@ public MetadataUpdate(
             @JsonProperty( SerializationConstants.PARTITIONS ) Optional<LinkedHashSet<Integer>> partitions,
             @JsonProperty( SerializationConstants.EXPIRATION ) Optional<DataExpiration> dataExpiration ) {
         // WARNING These checks have to be consistent with the same check elsewhere.
-        Preconditions.checkArgument( !title.isPresent() || StringUtils.isNotBlank( title.get() ),
+        Preconditions.checkArgument( title.isEmpty() || StringUtils.isNotBlank( title.get() ),
                 "Title cannot be blank." );
-        Preconditions.checkArgument( !name.isPresent() || StringUtils.isNotBlank( name.get() ),
+        Preconditions.checkArgument( name.isEmpty() || StringUtils.isNotBlank( name.get() ),
                 "Entity set name cannot be blank." );
-        Preconditions.checkArgument( !contacts.isPresent() || !contacts.get().isEmpty(), "Contacts cannot be blank." );
-        Preconditions.checkArgument( !type.isPresent() || StringUtils.isNotBlank( type.get().getNamespace() ),
+        Preconditions.checkArgument( contacts.isEmpty() || !contacts.get().isEmpty(), "Contacts cannot be blank." );
+        Preconditions.checkArgument( type.isEmpty() || StringUtils.isNotBlank( type.get().getNamespace() ),
                 "Namespace of type is missing." );
-        Preconditions.checkArgument( !type.isPresent() || StringUtils.isNotBlank( type.get().getName() ),
+        Preconditions.checkArgument( type.isEmpty() || StringUtils.isNotBlank( type.get().getName() ),
```

Comment:
Another way I've been cleaning these up lately is using `require`, `requireNotNull`, `check`, and `checkNotNull`. Lets you perform an action before the exception is thrown so you can `logger` before it gets thrown. Also feels cleaner than `Preconditions.checkArgument`

Reply:
I thought that was only available in Kotlin

Reply:
Oh derrrp. Yep didn't realize this was Java